src/main/kotlin/org/hyacinthbots/lilybot/utils/_Utils.kt

```diff
@@ -268,6 +268,80 @@ suspend inline fun CheckContext<*>.configPresent(vararg configOptions: ConfigOpt
 	}
 }
 
+suspend inline fun configIsUsable(option: ConfigOptions, guildId: Snowflake): Boolean {
+	when (option) {
+		ConfigOptions.SUPPORT_ENABLED -> {
+			val supportConfig = SupportConfigCollection().getConfig(guildId) ?: return false
+			return supportConfig.enabled
```

Comment:
These can probably be put into one line. 
`` SupportConfigCollection().getConfig(guildId).enabled ?: false ``